styles/components/categories.css

```diff
@@ -0,0 +1,66 @@
+.categories-section {
+  padding: 2rem;
+}
+.categories-section h2 {
+  text-align: center;
+  margin-bottom: 2rem;
+  color: var(--COLOR-LIGHT);
+}
+.categories-cont {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 1rem;
+}
+
+.category-card {
+  height: 300px;
+  border-radius: 1rem;
+  overflow: hidden;
+  background-color: var(--COLOR-LIGHT);
+}
+.category-card img {
+  width: 100%;
+  height: 60%;
+}
+.category-card img:hover {
+  transform: scale(1.1);
+  transition: transform ease 1s;
+}
+.category-body {
+  padding: 0.5rem 1rem;
+  position: relative;
+}
+.category-body .level {
```

Comment:
Everything is very nice especially the UI. Just one suggestion to add space after every css ruleset.